src/decoders/arw.rs

```diff
@@ -73,7 +73,7 @@ impl<'a> Decoder for ArwDecoder<'a> {
 }
 
 impl<'a> ArwDecoder<'a> {
-  fn image_a100(&self, camera: &Camera) -> Result<RawImage,String> {
```

Comment:
I can see one potential problem with this (maybe). The lifetime of these objects are tied to the lifetime of `RawLoader`, which makes sense when they were only tied to the camera toml file. Now they are based on orientation as well, which is image specific and still would have the life time of the `RawLoader` instance that created it. Wouldn't this mean a `RawLoader` would leak memory?